framework/mesh/mesh.h

```diff
@@ -90,44 +90,6 @@ ComputeLBF(std::vector<Vector3>& points, std::vector<double>& x_cuts, std::vecto
 /** Decomposes a 2D surface mesh using the centroids in a Px-Py fashion.*/
 void DecomposeSurfaceMeshPxPy(const SurfaceMesh& smesh, int Px, int Py);
```

Comment:
is this (`DecomposeSurfaceMeshPxPy`) still used?

Reply:
No, that is dead code as well. Do you want me to remove it as well?

Reply:
The code was removed but the declaration stayed. The same is true for `ComputeLBF`, so I will remove that as well...